### DISCUSSION TITLE
Link to database backup

### DISCUSSION THREAD

**Discussion by jelle-burodeng** [ExpressionEngine/ExpressionEngine#2958] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
When running the database backup utility, provide a download link to the just generated backup

**Describe the solution you'd like**
Not just displaying the name/path of the file, but just a clickable link

**Comment by intoeetive** [comment]
I believe the reason we don't do this is that on the setups where `system` folder is above web root the directory with backups is not accessible over HTTP.

So to achieve this we'd first need to add some methods to serve the files from protected folders via ACT requests

**Comment by jelle-burodeng** [comment]
True. so something like Link Vault
